daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -837,7 +837,9 @@ case class ParserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
     roundTrip: RoundTrip,
     implString: Option[String]) = {
 
-    val nBits = optLengthLimitInBits.get
+    val nBits = optLengthLimitInBits.getOrElse{
+      Assert.invariantFailed("TDML tests should always have a length limit.")
```

Comment:
Change to a plain 
`Assert.usage(optLengthLimitInBits.isDefined)` will make the coverage error go away.